### PR TITLE
add HELM support for aws-api-throttle

### PR DESCRIPTION
**Pull request by M00nF1sh**
### Issue

N/A

### Description

Add helm support for `aws-api-throttle`.

There is two style to supply it:
* ```helm upgrade aws-load-balancer-controller ./helm/aws-load-balancer-controller -n kube-system --set clusterName=some-cluster --set serviceAccount.create=false --set serviceAccount.name=aws-load-balancer-controller --set awsApiThrottle="{Elastic Load Balancing v2:RegisterTargets|DeregisterTargets=4:20,Elastic Load Balancing v2:.*=10:40}" ```
* ```helm upgrade aws-load-balancer-controller ./helm/aws-load-balancer-controller -n kube-system --set clusterName=some-cluster --set serviceAccount.create=false --set serviceAccount.name=aws-load-balancer-controller --set awsApiThrottle="Elastic Load Balancing v2:RegisterTargets|DeregisterTargets=4:20\,Elastic Load Balancing v2:.*=10:40" ``` (note: the `,` needs to be escaped with `\` when using this format)

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: helm/aws-load-balancer-controller/values.yaml [hunk 1]
```diff
@@ -140,9 +140,15 @@ vpcId:
 # Custom AWS API Endpoints (serviceID1=URL1,serviceID2=URL2)
 awsApiEndpoints:
 
+# Custom AWS API throttle settings (serviceID1:operationRegex1=rate:burst,serviceID2:operationRegex2=rate:burst)
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/values.yaml:143]: Lets specify the comments in this format:
\# awsApiThrottle specifies custom AWS API ...

Comment (kishorj) [reply helm/aws-load-balancer-controller/values.yaml:143]: Also, could we have some examples in the comments?

Comment (M00nF1sh) [reply helm/aws-load-balancer-controller/values.yaml]: make sense

**Review by kishorj — APPROVED** [review]

**Pull request merged**